app/src/main/java/com/cornellappdev/transit/models/RouteRepository.kt

```diff
@@ -15,8 +17,7 @@ import javax.inject.Singleton
  * Repository for data related to routes
  */
 @Singleton
-
-class RouteRepository @Inject constructor(private val networkApi: NetworkApi) {
```

Comment:
Hm, I see you changed this to a viewmodel, perhaps to use `viewModelScope`? Was there any particular reason/need to do this?

Reply:
I'm not sure either because I don't remember changing this. Maybe it was because I made this branch along with my previous one and accidentally committed to this one. I'll change it back!